startup.py

```diff
@@ -81,13 +81,11 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
         # note: all the business logic for how to launch is
         #       located in the python/startup folder to be compatible
         #       with older versions of the launch workflow
-        bootstrap_python_path = os.path.join(
-            self.disk_location, "python", "startup", "bootstrap.py"
+        bootstrap_python_path = os.path.join(self.disk_location, "python", "startup")
+        (file_obj, filename, desc) = imp.find_module(
+            "bootstrap", [bootstrap_python_path]
         )
-        with open(bootstrap_python_path, "r") as mod_file:
-            bootstrap = imp.load_module(
-                "bootstrap", mod_file, "tk-photoshopcc.bootstrap", (".py", "U", 1)
-            )
+        bootstrap = imp.load_module("bootstrap", file_obj, filename, desc)
```

Comment:
This was erroring on Python 3 as it couldn't find the filename that it was passed.
I've updated it to get the details from find_module, but this is still using the deprecated `imp`. 

I did consider sys.path.insert and then removing the path after importing, as the alternatives to `imp` don't appear to be that backwards compatible.

Reply:
Could you use the `sys.path.insert` method instead, so we don't have to revisit this when `imp` goes away?

Reply:
As discussed I've duplicated up our internal core imp ticket and made it about Photoshop.